query.go

```diff
@@ -105,9 +105,9 @@ func (c *Client) QueryChannels(q *QueryOption, sort ...*SortOption) ([]*Channel,
 
 type SearchRequest struct {
 	// Required
-	Query   string                 `json:"query"`
-	Filters map[string]interface{} `json:"filter_conditions"`
-
+	Query          string                 `json:"query"`
+	Filters        map[string]interface{} `json:"filter_conditions"`
+	MessageFilters map[string]interface{} `json:"message_filters"`
```

Comment:
are message filters required or optional?

Reply:
good point. At least one of the fields is required. Validator.v9 has troubles with cross-field validation, so I've added checking inside the code.